### ISSUE TITLE
Hunterborn Collection to preserve token looting not working

### ISSUE THREAD

**Issue by SteveTownsend** [SteveTownsend/SmartHarvestSE#87]
Nexus report.

**Comment by SteveTownsend** [comment]
sent this protoype for testing by reporting user:
```
{
  "$comment": "Definitions are allowed for up to 128 rulesets for Collections",
  "$schema": "./SHSE.SchemaCollections.json",
  "collections": [
    {
      "name": "Hunterborn Tokens",
      "description": "Prevent SHSE from messing up Hunterborn Tokens",
      "policy": {
        "action": "leave",
        "notify": false,
        "repeat": false
      },
      "rootFilter": {
        "operator": "AND",
        "condition": {
          "plugin": [ "Hunterborn.esp" ],
          "keyword": [ "_DS_KW_MonsterToken", "_DS_KW_AnimalToken", "_DS_KW_Token" ]
        }
      }
    }
  ]
}
```

**Comment by SteveTownsend** [comment]
Shipped this with v1.5.0.1
